plugins/source/aws/docs/tables/aws_cloudwatchlogs_metric_filters.md

```diff
@@ -2,7 +2,7 @@
 
 https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_MetricFilter.html
 
-The primary key for this table is **arn**.
+The composite primary key for this table is (**arn**, **log_group_name**).
```

Comment:
I don't like the custom/artificial `arn` here, maybe change it to be account+region+metric name+log group name?

Reply:
done in [`59b19a1` (#8061)](https://github.com/cloudquery/cloudquery/pull/8061/commits/59b19a18f618a7c2907736e77a5f3145c68da3bb)